src/main/java/org/myrobotlab/swing/widget/DockableTab.java

```diff
@@ -232,10 +233,17 @@ public void run() {
         undocked.getContentPane().add(display);
 
         // icon
-        URL url = getClass().getResource(Util.getResourceDir() + "/mrl_logo_36_36.png");
-        Toolkit kit = Toolkit.getDefaultToolkit();
-        Image img = kit.createImage(url);
-        undocked.setIconImage(img);
+        // URL url = getClass().getResource(Util.getResourceDir() +
+        // "/mrl_logo_36_36.png");
+        URL url = null;
+        try {
+          url = new URL("file:///" + Util.getResourceDir() + File.separator + "mrl_logo_36_36.png");

```

Comment:
hmm.. i think you want to getResourceDir here with forward slashes, and not the system dependant ones..

Reply:
heh .. your right .. I first switched that one .. and it did not work ..
only after did I add the file:/// reference .. changing it back to forward slash ..